docs/readmes/dp/debug_logs.md

```diff
@@ -7,7 +7,111 @@ id: debug_logs title: View Logs hide_title: true
+    1. `2022-04-25 15:41:17,550 DEBUG (Inform msg) Received parameter: Device.DeviceInfo.HardwareVersion = ...`
+    1. `[Inform msg output continues]`
+    1. `2022-04-25 15:41:17,557 DEBUG State transition from <WaitInformState> to <wait_empty>`
+    1. `2022-04-25 15:41:17,558 DEBUG Sending TR069 message: InformResponse {'MaxEnvelopes': '1'}`
+1. Sequence of `GetParameterValues` messages (Transient, Normal, Object)
+    1. `2022-04-25 15:41:22,608 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 419`
+    1. `2022-04-25 15:41:22,612 DEBUG Handling TR069 message: DummyInput {}`
+    1. `2022-04-25 15:41:22,612 DEBUG State transition from <WaitEmptyMessageState> to <check_fw_upgrade_download>`
+    1. `12022-04-25 15:41:22,613 DEBUG Skipping FW Download for eNB [XXXXXXXXXXXXX], missing firmware upgrade config in enodebd.yml.`
+    1. `2022-04-25 15:41:22,613 DEBUG State transition from <CheckFirmwareUpgradeDownloadState> to <get_transient_params>`
+    1. `2022-04-25 15:41:22,613 DEBUG State transition from <SendGetTransientParametersState> to <wait_get_transient_params>`
+    1. `2022-04-25 15:41:22,613 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,617 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 912`
+    1. `2022-04-25 15:41:22,662 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+    1. `2022-04-25 15:41:22,662 DEBUG Fetched Transient Params: {...}`
+    1. `2022-04-25 15:41:22,663 DEBUG State transition from <WaitGetTransientParametersState> to <get_params>`
+    1. `2022-04-25 15:41:22,664 DEBUG State transition from <GetParametersState> to <wait_get_params>`
+    1. `2022-04-25 15:41:22,664 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,671 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 3242`
+    1. `2022-04-25 15:41:22,718 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+    1. `2022-04-25 15:41:22,719 DEBUG Received CPE parameter values: {...}`
+    1. `2022-04-25 15:41:22,719 DEBUG State transition from <WaitGetParametersState> to <get_obj_params>`
+    1. `2022-04-25 15:41:22,720 DEBUG State transition from <GetObjectParametersState> to <wait_get_obj_params>`
+    1. `2022-04-25 15:41:22,720 DEBUG Sending TR069 message: GetParameterValues {...}`
+    1. `2022-04-25 15:41:22,723 DEBUG 10.0.2.243 - "POST / HTTP/1.1" 200 877`
+    1. `2022-04-25 15:41:22,769 DEBUG Handling TR069 message: GetParameterValuesResponse {...}`
+    1. `2022-04-25 15:41:22,770 DEBUG Received object parameters: {...}`
+
+1. `SetParameterValues` message followed by `GetParameterValues` - conditional, only appears if a configuration change is needed to be done on the eNB side
+(the actual parameters that are set in order to enable transmission based on Domain Proxy grant may differ depending on the eNB model and eNB current configuration state - please refer to your eNB device model implementation
+in `lte/gateway/python/magma/enodebd/devices`)
```

Comment:
These are actually 3 sentences. I.e.
- `'SetParameterValues' ... eNB side.` Full Stop.
- `The actual ... configuration state.` Full Stop.
- `Please refer to ... in XXX.` Full Stop.

Reply:
Corrections made.